app/controllers/homepage_controller.rb

```diff
@@ -0,0 +1,6 @@
+class HomepageController < ApplicationController
+  rescue_from GdsApi::HTTPForbidden, with: :error_403
```

Comment:
I think this will be useful when we start using the examples / calling the content store. If moving to a new controller we should _probably_ test this.